test/common/k8s/deploy/deploy.go

```diff
@@ -83,7 +126,7 @@ func Delete(name, templateOutputPath string) error {
 	util.PrintCommand(cmd)
 	_, err := cmd.CombinedOutput()
 	if err != nil {
-		return errors.Wrap(err, "Failed to delete AzureIdentityBinding from the Kubernetes cluster")
+		return errors.Wrapf(err, "Failed to delete %v from the Kubernetes cluster", name)
```

Comment:
Isn't this create failure ?

Reply:
no it's a delete operation (it's part of the `Delete()` func)